scripts/variantstore/beta_docs/run-your-own-samples.md

```diff
@@ -10,20 +10,41 @@ The [GVS workflow](https://github.com/broadinstitute/gatk/blob/ah_var_store/scri
 
 To learn more about the GVS workflow, see the [Genomic Variant Store workflow overview](./gvs-overview.md).
 
-### What does it require as input?
+## What does it require as input?
 
 - Reblocked single sample GVCF files with specific annotations described below
-- GVCF index files 
+- GVCF index files
 
-While the GVS has been tested with 250,000 single sample whole genome GVCF files as input, only datasets of up to 25,000 whole genomes are being used for beta testing.
+While the GVS has been tested with 410,000 single sample whole genome GVCF files and 190,000 single sample whole exome GVCF files as input, only datasets of up to 25,000 whole genomes and 100,000 whole exomes are currently supported by the beta workflow.
 
-#### Reblocked GVCF files
+### Supported data types
+_* indicates regularly tested and validated_
 
-If your GVCF files have not been reblocked, you can reblock them using the [WARP reblocking workflow](https://github.com/broadinstitute/warp/blob/master/pipelines/broad/dna_seq/germline/joint_genotyping/reblocking/ReblockGVCF.wdl), which is configured in the [ReblockGVCF Terra workspace](https://app.terra.bio/#workspaces/warp-pipelines/ReblockGVCF). 
+**Species**
+- Human data only (hg38)*
+
+**Sequencing details**
+- Illumina short read sequencing*
+- Whole Genome Sequencing*
+- Whole Exome Sequencing*
+- Blended Genome Exome Sequencing*
+
+**Secondary Analysis Methods**
+- DRAGEN 3.7.8*
```

Comment:
Is the star a footnote or do they have patch versions beyond 3.7.8?

Reply:
This maybe is confusing but the * is a .. headnote? I have it at the top of the list "* indicates regularly tested and validated" vs others without stars are things that we are aware have been run through the pipeline before and expect it will work.